### ISSUE TITLE
Staging Review finding: Progress bar text not appropriate text for a heading

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#104821]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Ask VA Integration
**Product:** Ask VA
**Feature:** Ask Va

## Findings details
**VA.gov Experience Standard - issue:** [User encounters consistent, clear design components and patterns.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

Headings are often used by screen reader users to quickly navigate a page. They function like labels for sections, and skipping heading-to-heading tells you what content to expect in that section of the page.

On a typical form on VA.gov, the pages in each step of the form flow have a step indicator that identifies the step numerically and describes the purpose of that step. You end up with a page structure like:
- H1: Apply for some benefit
   - H2: Step 1 - Personal information
      - H3: Your contact information

The forms library automatically sends focus to the step indicator H2, so as soon as I land on the page I know what section of the form I'm in. Note that the H2 isn't just "Step 1," but also contains a concise description of what's being asked of you in this section of the form.

With the progress bar in place of the step indicator, each page of the form flow is instead labeled with an H2 heading that indicates how much of the form has been completed so far --- but doesn't tell you anything about what's on that page or what the purpose of this part of the form flow is.

I think this is a meaningful regression, and behavior that may be confusing to screen reader users who are used to the content labeling that occurs with the step indicator on other forms. I don't think it will block anyone from moving through the form, however.

<details><summary>Link, screenshot or steps to recreate</summary>

![The top of a page in the Ask VA form flow, with the progress bar heading reading "13% complete with form"](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/62304-Brian-2.png?raw=true)

</details>

## Recommended action

I don't have a concrete recommendation to make, but I would encourage you to explore including a concise description of the section of the form that you're working through in the heading. I think it would be fine for this description to span multiple pages in the flow so long as it's still an appropriate description (eg. "5% complete - Your information", "13% complete - Your information" etc), so long as it provides some meaningful information in addition to just progress.

Alternatively, consider reaching out to the forms team for their work on the [minimal header page structure for forms](https://staging.va.gov/mock-form-minimal-header/name-and-date-of-birth), which removes the step indicator from the heading hierarchy and instead focuses on the question shown on the page as an H1. There are a number of ways in which this is a better pattern than what's the default in the forms library, and following a similar structure while using the progress bar would sidestep needing to add any extra text to the progress bar (although it's definitely a heavier technical lift).

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
